crates/sui-indexer-alt-framework/src/pipeline/concurrent/collector.rs

```diff
@@ -194,12 +194,14 @@ pub(super) fn collector<H: Handler + 'static>(
 }
 
 #[cfg(test)]
+#[cfg(feature = "postgres")]
```

Comment:
This will cause these tests to be disabled if the `postgres` feature is not provided. That doesn't seem right. Instead, I think you want to require `sui-pg-db` is a `dev-dependency` of the framework, so that even if the crate is configured to build without postgres support, we can have that support to build and run tests.

(You may also need to change where you are importing things like `DbConnection` and `FieldCount` from, so that they are fetched directly from `sui_pg_db`, and not via the exports that you set up if the `postgres` feature is enabled).